script/pre_immersive_distorted.py

```diff
@@ -329,24 +330,21 @@ def imageundistort(video, offsetlist=[0],focalscale=1.0, fixfocal=None):
 
 
 
-def softlinkdataset(originalpath, path, srcscene, scene):
-    videofolderlist = glob.glob(originalpath + "camera_*/")
-
-    if not os.path.exists(path):
-        os.makedirs(path)
+def softlinkdataset(original_str, target_str):
+    originalpath = Path(original_str)
+    path = Path(target_str)
 
+    videofolderlist = [f for f in sorted(originalpath.glob("camera_*")) if f.is_dir()]
```

Comment:
as _pathlib.glob_ ignored "/" (contrary to _glob_ lib) adding f.is_dir() check is needed. https://stackoverflow.com/a/76708735/352173